tools/extract-plugin-docs.js

```diff
@@ -23,14 +23,27 @@ var pluginCats = Object.keys(plugins)
 
 var docsPath = 'documentation/browser/plugins'
 
-if (process.argv.indexOf('-f') === -1) {
-	console.log(`usage: extract-plugin-docs -f
+if (process.argv.indexOf('-f') === -1 && process.argv.indexOf('-o') === -1) {
```

Comment:
please look at using `minimist` for this 
https://github.com/substack/minimist
https://github.com/vizorvr/vizor/blob/develop/tools/upload.js#L9